include/saber/inexact.hpp

```diff
@@ -0,0 +1,71 @@
+#ifndef SABER_INEXACT_HPP
+#define SABER_INEXACT_HPP
+#pragma once
+
+// std
+#include <algorithm>
+#include <cmath>
+#include <limits>
+#include <cassert>
+#include <type_traits>
+
+namespace saber {
+
+class Inexact
+{
+public:
+    template<typename T>
+    static bool IsEQ(T inLHS, T inRHS)
+    {
+        EQ equal{inLHS};
+        const bool result = equal(inRHS);
+        return result;
+    }
+
+    /*
+    template<typename T>
+    static bool IsNE(T inLHS, T inRHS)
+    {
+        EQ unequal{inLHS};
```

Comment:
`IsNE()` should be using the functor `Ne`.
While the functor `Ne` should use the result of `!Eq()`